index.js

```diff
@@ -544,82 +557,4 @@ module.exports = {
     "template-curly-spacing": ["error", "never"],
     "yield-star-spacing": ["error", "after"],
   },
-  overrides: [
```

Comment:
Pulled all ts stuff out into a separate file, so you don't need the ts dependencies if you're not going to use them.